### PR TITLE
feature/CXMC-22861: Implement login out in spa

### PR DESCRIPTION
**Pull request by Melody-zhou-512**
JIRA ticket:https://jira.tools.sap/browse/CXMC-22861

### CODE REVIEW

File: projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts [hunk 1]
```diff
@@ -4,30 +4,30 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { NgModule } from '@angular/core';
-import { CdcConfig, CdcRootModule, CDC_FEATURE } from '@spartacus/cdc/root';
-import { CmsConfig, I18nConfig, provideConfig } from '@spartacus/core';
+import {  NgModule } from '@angular/core';
```
Comment (i53577) [inline projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts]: please format code

File: projects/storefrontapp/src/app/spartacus/features/cdc/cdc-config-initializer.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+export class CdcConfigInitializer implements ConfigInitializer {
+  readonly scopes = ['cdc'];
+  readonly configFactory = () => lastValueFrom(this.resolveConfig());
+  protected baseSiteService = inject(BaseSiteService);
+  /**
+   * Emits the Cdc config basing on the current base site data.
+   *
+   * Completes after emitting the value.
+   */
+  protected resolveConfig(): Observable<CdcConfig> {
```
Comment (i53577) [inline projects/storefrontapp/src/app/spartacus/features/cdc/cdc-config-initializer.ts]: what about move these logic to /spartacus/integration-libs/cdc/root/config ?

File: projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts (hunk 1 above)
Comment (Melody-zhou-512) [reply projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts]: have been modified.

File: projects/storefrontapp/src/app/spartacus/features/cdc/cdc-config-initializer.ts (hunk 2 above)
Comment (Melody-zhou-512) [reply projects/storefrontapp/src/app/spartacus/features/cdc/cdc-config-initializer.ts]: have been modified.

File: projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts [hunk 3]
```diff
@@ -5,29 +5,37 @@
...
-          sessionExpiration: 3600,
-        },
-        {
-          baseSite: 'powertools-spa',
-          javascriptUrl: 'JS_SDK_URL_PLACEHOLDER',
-          sessionExpiration: 3600,
-        },
-      ],
-    }),
+    {
```
Comment (vinchen666) [inline projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts]: let's move to root module

Comment (Melody-zhou-512) [reply projects/storefrontapp/src/app/spartacus/features/cdc/cdc-feature.module.ts]: have been modified.

**Review by vinchen666 — APPROVED** [review]

**Pull request merged**